lib/ansible/utils/collection_loader/_collection_finder.py

```diff
@@ -1133,16 +1133,7 @@ def _get_collection_playbook_path(playbook):
 
 
 def _get_collection_role_path(role_name, collection_list=None):
-    return _get_collection_resource_path(role_name, u'role', collection_list)
-
-
-def _get_collection_resource_path(name, ref_type, collection_list=None):
```

Comment:
don't delete this function, we use it for more than roles, it is generic for a reason